### PR TITLE
Add HTTP transport example first in the README.

### PR DESCRIPTION
**Pull request by mjbryant**

### CODE REVIEW

**Comment by mjbryant** [comment]
Addresses #11

**Comment by mjbryant** [comment]
@adriancole want to take a look? How I encode the span in a list is kinda hacky, but the collector wasn't accepting non-list values.

File: README.md [hunk 1]
```diff
@@ -109,8 +109,30 @@ Transport
...
+argument - the thrift-encoded bytes.
+
+The simplest way to get spans to the collector is via HTTP POST. Here's an
+example of a simple HTTP transport using the `requests` library. This assumes
+your Zipkin collector is running at localhost:9411.
+
+```python
+import requests
+
+def http_transport(message):
```
Comment (adriancole) [inline README.md]: Nit since the input is a single span, I'd probably name it encoded_span or just span. In zipkin we call the payload sent message

**Comment by adriancole** [comment]
Thanks! This taught me that the input is a single span (vs a "bundle" aka
list of them). Somehow I guessed the input was a list already, but yeah I
can see how someone can write a async/batch handler.

File: README.md (hunk 1 above)
Comment (luukhoai) [inline README.md]: Hi, I checked this code above. However, I didn't see where to input service_name & span_name.
It also not working. Thanks

Comment (luukhoai) [inline README.md]: screenshot from 2016-10-12 08 15 10

Hi, I checked. but it raise new error: py_zipkin.exception.ZipkinError: Root spans require a transport handler to be given. 
Please check, thanks so much :D

Comment (kaisen) [inline README.md]: the transport handler is supposed to be a function, so try passing in http_transport as the parameter value instead.

transport_handler=http_transport,

Comment (luukhoai) [inline README.md]: localhost:9411 also not working too. Thanks

**Comment by mjbryant** [comment]
@adriancole yeah, our transport handler just takes a single span. The user can implement batching. I still want to implement #6 to change this to a list of spans, though. Seems strictly more flexible and should make implementing a batched transport handler a bit easier.

File: README.md (hunk 1 above)
Comment (luukhoai) [inline README.md]: It's working. THank you so much :D

**Comment by adriancole** [comment]
I made an example out of this. if you don't mind, please take a look! https://github.com/openzipkin/pyramid_zipkin-example/pull/1

**Pull request merged**